### ISSUE TITLE
aws_codebuild_project is missing a valid container type (WINDOWS_CONTAINER)

### ISSUE THREAD

**Issue by ryno75** [terraform-providers/terraform-provider-aws#4940]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or "me too" comments, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Description

The aws_codebuild_project resource `enviroinment` property `type` currently only supports a value of `LINUX_CONTAINER` but `WINDOWS_CONTAINER` should be a valid property (it is supported by aws-sdk-go).

The code in v1.24.0 that this applies to is:
```go
155                         "type": {
156                             Type:     schema.TypeString,
157                             Required: true,
158                             ValidateFunc: validation.StringInSlice([]string{
159                                 codebuild.EnvironmentTypeLinuxContainer,
160                             }, false),
161                         },
```

### New or Affected Resource(s)

* aws_codebuild_project

### Potential Terraform Configuration

```hcl
resource "aws_codebuild_project" "test" {
  name           = "${local.codebuild_proj_name}"
  description    = "test_codebuild_project"
  build_timeout  = "5"
  service_role   = "${aws_iam_role.codebuild.arn}"
  encryption_key = "arn:aws:kms:${var.aws_region}:${data.aws_caller_identity.current.account_id}:alias/aws/s3"

  artifacts {
    type = "CODEPIPELINE"
    name = "test-bld"
  }

  environment {
    compute_type    = "${local.codebuild_compute_type}"
    image           = "${local.codebuild_docker_image}"
    type            = "WINDOWS_CONTAINER"
    privileged_mode = false

  }

  source {
    type = "CODEPIPELINE"
  }
}

```

**Comment by callmeradical** [comment]
👍 I would love to see this

**Comment by bflad** [comment]
Support for `WINDOWS_CONTAINER` has been merged into master via #4960 and will release with version 1.25.0 of the AWS provider, likely middle of next week.

**Comment by bflad** [comment]
This has been released in [version 1.25.0 of the AWS provider](https://github.com/terraform-providers/terraform-provider-aws/blob/master/CHANGELOG.md#1250-june-27-2018). Please see the [Terraform documentation on provider versioning](https://www.terraform.io/docs/configuration/providers.html#provider-versions) or reach out if you need any assistance upgrading.
